packages/@aws-cdk/aws-iam/lib/policy-statement.ts

```diff
@@ -239,10 +260,17 @@ export interface PolicyStatementProps {
   /**
    * Resource ARNs to add to the statement
    *
-   * @default - no principals
+   * @default - no resources
    */
   readonly resources?: string[];
 
+  /**
+   * NotResource ARNs to add to the statement
+   *
+   * @default - no resources
+   */
+  readonly notresources?: string[];
```

Comment:
Shouldn't this be called `notResources`? Camel case?

Reply:
You are right, it should have been.

Reply:
You want to change this before v1.5.0?

Reply:
Hell yes :)

Reply:
I can quickly open a PR to fix the casing, The check against combining `actions` and `notActions` can wait after v1.5.0 I presume.

Reply:
OK you did it already :)